templates/upstart.conf.j2

```diff
@@ -0,0 +1,14 @@
+description "Gunicorn upstart script for {{ gunicorn_app_name }}"
+
+{% if develop or test -%}
```

Comment:
Maybe instead of using upstart, we should just use gunicorn's init with `/etc/gunicorn.d/`?